app/api/v2/models/orders_model.py

```diff
@@ -0,0 +1,126 @@
+from app.db_config import db_init
+
+
+class OrderModel(object):
+    """"Class to handle admin models."""
+
+    def __init__(self):
+        self.db = db_init()
+
+    def get_all_orders(self):
+        """"Method to fetch all parcel order deliveries."""
+
+        self.db = db_init()
+        cur = self.db.cursor()
+        cur.execute(
+            """SELECT order_id, user_id, pickup_location, destination, weight, price, status, date_created  FROM orders""")
+        rows = cur.fetchall()
+        self.db.close()
+        data = []
+        for i, items in enumerate(rows):
+            order_id, user_id, pickup_location, destination, weight, price, status, date_created = items
+            users = dict(order_id=order_id, user_id=user_id, pickup_location=pickup_location,
+                         destination=destination, weight=weight, price=price, status=status, date_created=date_created)
+            data.append(users)
+        return data
+
+    def get_one_order(self, parcelId):
+        """"Method to fetch one parcel order delivery"""
+
+        self.db = db_init()
+        cur = self.db.cursor()
+        cur.execute("""SELECT user_id, user_id, pickup_location, destination, weight, price, status, date_created  FROM orders WHERE order_id = %s""", (parcelId, ))
+        rows = cur.fetchall()
+        self.db.close()
+        data = []
```

Comment:
I think the `for loop` can be condensed into a method to avoid repetition since you have used it several times